warehouse/oidc/models/activestate.py

```diff
@@ -122,7 +122,7 @@ def publisher_name(self) -> str:
     def project(self) -> str:
         return self.activestate_project_name
 
-    def publisher_url(self, claims: SignedClaims | None = None) -> str:
+    def publisher_url(self, claims=None):
```

Comment:
Curious about why I removed the type declarations?  Or why claims are passed to this function at all?  

The former I did because, at the time, `oidc.models._core.OIDCPublisherMixin.publisher_url` didn't have the types.  Now that I look again though, they DO have them, though slightly different.  I'll bring them back.

The latter:  To fulfill the "interface" of `OIDCPublisherMixin` and override `pubslisher_url` there.  I could technically leave it out but we could run into issues some day if someone is iterating over publishers and calling their `publisher_url` and it fails on the ActiveState publisher because `publisher_url` only takes `self` and the calling code is trying to pass it `claims`.

I'll push a changes to bring back the typing.  Let me know if my explanation of including `claims` as a method arg makes sense.